packages/definitions-parser/src/git.ts

```diff
@@ -148,11 +174,8 @@ it is supposed to replace, ${typings.version} of ${unneeded.name}.`);
  */
 export async function getNotNeededPackages(
   allPackages: AllPackages,
-  diffs: GitDiff[],
+  deletions: PackageId[],
```

Comment:
At some level, it feels weird that a getter is also doing these kinds of checks, but not really for fixing in this PR.

Reply:
Yeah, the function is primarily "check the packages of these files" but it also returns a list of those packages that don't have errors.
Not sure changing the name would make it better, though, because then it would be a check- function that returns something.